docs/README.md

```diff
@@ -0,0 +1,58 @@
+# AlgoKit TypeScript AVM Debugging Utilities
```

Comment:
We have 2 readme files which essentially contain similar content. Probably no need?